src/templates/_common/components/payment_logo.jsx

```diff
@@ -5,19 +5,19 @@ const payment_methods_list = [
     { image: 'mastercard',              param: '?anchor=mastercard' },
     { image: 'bank_transfer',           param: '?anchor=bank-transfer' },
     { image: 'internet_bank_transfer',  param: '?anchor=internet-bank-transfer' },
-    { image: 'paysec',                  param: '?anchor=paysec',         className: 'eu-hide' },
+    { image: 'paysec',                  param: '?anchor=paysec',         dataShow: '-malta, -maltainvest, -iom, -country:mt' },
```

Comment:
does `country:mt` works only for all eu countries?

Reply:
@kellybinary In here only `mt` is assumed as an EU country:
 https://github.com/binary-com/binary-static/blob/f7415ed9a834234919bc23a4cd20d7618ace30e1/src/javascript/app/common/country_base.js#L10
Are there other countries?

Reply:
no, only MT is an EU country which does not have landing company as EU since we don't offer account opening there.